dune.opam

```diff
@@ -40,7 +40,7 @@ build: [
   # opam 2 sets OPAM_SWITCH_PREFIX, so we don't need a hardcoded path
   ["ocaml" "configure.ml" "--libdir" lib] {opam-version < "2"}
   ["ocaml" "bootstrap.ml" "-j" jobs]
-  ["./dune.exe" "build" "-p" name "--profile" "dune-bootstrap" "-j" jobs]
+  ["./dune.exe" "build" "--release" "--profile" "dune-bootstrap" "-j" jobs]
```

Comment:
We need to pass `dune.install` explicitly, otherwise it's going to build `@install` which is going to build all the packages, not just dune.

Reply:
Good point. Fixed.